<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="de_CH">
<context>
    <name>EditableGeoCsv</name>
    <message>
        <location filename="geocsv_plugin.py" line="57"/>
        <source>Add GeoCSV layer</source>
        <translation>GeoCSV Layer hinzufügen</translation>
    </message>
    <message>
        <location filename="geocsv_plugin.py" line="64"/>
        <source>Editable GeoCSV</source>
        <translation>Editable GeoCSV</translation>
    </message>
</context>
<context>
    <name>GeoCsvNewController</name>
    <message>
        <location filename="geocsv_controller.py" line="115"/>
        <source>invalid file path</source>
        <translation>Der angegebene Pfad ist ungültig</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="174"/>
        <source>unknown csvt attribute: {}</source>
        <translation>Ungültiges csvt Attribut: {}</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="176"/>
        <source>csv&lt;-&gt;csvt missmatch</source>
        <translation>Das CSVT-Schema passt nicht zum angegeben CSV-File</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="178"/>
        <source>csvt geometry type exception</source>
        <translation>Falsche Geometrieangaben</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="180"/>
        <source>no csvt file found</source>
        <translation>Kein CSVT-File gefunden</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="204"/>
        <source>error while loading csv</source>
        <translation type="unfinished">Fehler beim Laden des CSV-Files</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="194"/>
        <source>error in geometry selection</source>
        <translation>Fehler in den Geometrieangaben</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="77"/>
        <source>GeoCSV Layer created</source>
        <translation>GeoCSV layer erstellt</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="77"/>
        <source>The layer &quot;{}&quot; was created successfully.</source>
        <translation>Der layer &quot;{}&quot; wurde erfolgreich erstellt</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="134"/>
        <source>Open GeoCSV File</source>
        <translation>GeoCSV Dokument öffnen</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="305"/>
        <source>CSVT File Error</source>
        <translation>Fehler in CSVT File</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="83"/>
        <source>The csvt file couldn&apos;t be updated on disk.</source>
        <translation>Das CSVT File konnte nicht aktualisiert werden.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="305"/>
        <source>An error occured while trying to update the CSVT file according to the new attribute types. Please update the csvt file manually.</source>
        <translation>Fehler beim Aktualisieren des CSVT Files. Bitte aktualisieren Sie das CSVT File manuell.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="140"/>
        <source>not utf8 encoded</source>
        <translation>Das csv File ist nicht UTF-8 kodiert.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="155"/>
        <source>error in converting</source>
        <translation>Die Konvertierung ist Fehlgeschlagen. Bitte manuell in UTF-8 kodieren.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="150"/>
        <source>backup created</source>
        <translation>Backup erstellt</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="150"/>
        <source>Created backup on &quot;{}&quot;</source>
        <translation>Backup unter &quot;{}&quot; erstellt. </translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="152"/>
        <source>Converted</source>
        <translation>Konvertiert</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="152"/>
        <source>Successfully converted to utf-8</source>
        <translation>CSV wurde erfolgreich nach utf-8 konvertiert.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="134"/>
        <source>Files (*.csv *.tsv *.*)</source>
        <translation>Dokumente (*.csv *.tsv *.*)</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="119"/>
        <source>invalid delimiter. expected &quot;{}&quot;</source>
        <translation>Fehlerhaftes Trennzeichen in CSV Dokument. Laut GeoCSV-Spec wird ein &quot;{}&quot; erwartet.</translation>
    </message>
</context>
<context>
    <name>GeoCsvReconnectController</name>
    <message>
        <location filename="geocsv_controller.py" line="280"/>
        <source>Couldn&apos;t automatically restore csv layer &quot;{}&quot;</source>
        <translation>Fehlende Angaben für den GeoCSV-Layer {}</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="278"/>
        <source>GeoCSV Layer reconnected</source>
        <translation>GeoCSV Layer erfolgreich geladen</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="278"/>
        <source>Layer &quot;{}&quot; is successfully reconnected</source>
        <translation>Der Layer &quot;{}&quot; wurde erfolgreich geladen</translation>
    </message>
</context>
<context>
    <name>VectorLayerController</name>
    <message>
        <location filename="geocsv_controller.py" line="292"/>
        <source>CSV File updated</source>
        <translation>CSV Dokument aktualisiert</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="292"/>
        <source>Changes to layer &quot;{}&quot; successfully stored in csv file.</source>
        <translation>Änderungen im layer &quot;{}&quot; wurden erfolgreich gespeichert. </translation>
    </message>
</context>
<context>
    <name>VectorLayerSaveConflictController</name>
    <message>
        <location filename="geocsv_controller.py" line="368"/>
        <source>Invalid path</source>
        <translation>Der angegebene Pfad ist ungültig</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="368"/>
        <source>An error occured while trying to save file on new location. Please try again.</source>
        <translation>Beim Versuch, das Dokument am neuen Ort zu speichern, ist ein Fehler aufgetreten. Bitte versuchen Sie es erneut.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="317"/>
        <source>CSVT file could not be updated</source>
        <translation>CSVT Dokument konnte nicht aktualisiert werden</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="317"/>
        <source>An error occured while trying to update the CSVT file according to the new attribute types. Please update the csvt file manually.</source>
        <translation>Fehler beim Aktualisieren des CSVT Files. Bitte aktualisieren Sie das CSVT File manuell.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="312"/>
        <source>Error while updating attributes happend</source>
        <translation>Fehler beim Aktualisieren der Attribute aufgetreten</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="312"/>
        <source>An error occured while trying to update the attributes list. Nothing has been stored on disk.</source>
        <translation>Beim Versuch, die Attributliste zu aktualisieren, ist ein Fehler aufgetreten. Es wurde nichts gespeichert.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="322"/>
        <source>Geometry index violation</source>
        <translation>Fehler in den Geometrie-Angaben</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="322"/>
        <source>You tried to delete an attribute which is providing geometry information. The change will not be saved to disk.</source>
        <translation>Das gelöschte Attribut enthielt Geometrieangaben. Dies kann zu unbestimmten Problemen führen. Bitte verlassen Sie den Editiermodus ohne die Änderungen zu speichern.</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="366"/>
        <source>CSV File updated</source>
        <translation>CSV Dokument aktualisiert</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="366"/>
        <source>Changes to layer &quot;{}&quot; successfully stored in csv file.</source>
        <translation>Änderungen im layer &quot;{}&quot; wurden erfolgreich gespeichert. </translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="359"/>
        <source>Save File</source>
        <translation>Dokument speichern</translation>
    </message>
    <message>
        <location filename="geocsv_controller.py" line="359"/>
        <source>Files (*.csv *.tsv *.*)</source>
        <translation>Dokumente (*.csv *.tsv *.*)</translation>
    </message>
</context>
<context>
    <name>geocsvDialogNew</name>
    <message>
        <location filename="geocsv_dialog_new.ui" line="14"/>
        <source>GeoCSV Editor</source>
        <translation>Neuer GeoCSV Layer hinzufügen</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_conflict.ui" line="29"/>
        <source>Retry</source>
        <translation>Nochmals versuchen</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="68"/>
        <source>Cancel</source>
        <translation>Abbrechen</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_conflict.ui" line="43"/>
        <source>Save As</source>
        <translation>Speichern unter...</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_conflict.ui" line="59"/>
        <source>Error: The csv file could not be saved. File Lock?</source>
        <translation>Die Änderungen konnten nicht ins CSV-File gespeichert werden, da das File nicht gefunden oder gelocked ist. </translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="29"/>
        <source>Dateiname</source>
        <translation>Dateiname</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="46"/>
        <source>Durchsuchen...</source>
        <translation>Durchsuchen...</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="78"/>
        <source>Ok</source>
        <translation>OK</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="122"/>
        <source>WKT Field</source>
        <translation>WKT Feld</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="136"/>
        <source>Point (Easting/Northing)</source>
        <translation>Easting/Northing (Point)</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="155"/>
        <source>WKT</source>
        <translation>WKT (Point, LineString, Polygon)</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="200"/>
        <source>Easting Field</source>
        <translation>Easting Feld</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="213"/>
        <source>Northing Field</source>
        <translation>Northing Feld</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="242"/>
        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://giswiki.hsr.ch/Editable_GeoCSV#Help&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2b84d3;&quot;&gt;Help&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="270"/>
        <source>Convert</source>
        <translation>Konvertieren</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="277"/>
        <source>Character set:</source>
        <translation>Zeichensatz:</translation>
    </message>
    <message>
        <location filename="geocsv_dialog_new.ui" line="293"/>
        <source>The csv File is not in UTF-8. Please convert it first
 (CAUTION: the original file will be overwritten):</source>
        <translation>Das csv Dokument ist nicht UTF-8 kodiert. Bitte zuerst konvertieren. (ACHTUNG: Das Originaldokument wird dabei überschrieben; es wird ein Backup erstellt.)</translation>
    </message>
</context>
</TS>
